pages/en/partials/class-electronic.md

```diff
@@ -6,3 +6,100 @@ readMore:   class-electronic
+- Understand how to read a basic electronic circuit diagram
+- Creating basic circuits virtually using software tools
+- Build circuits in teams using a breadboard, jumper cables and a variety of components
+- Understanding voltage / current / resistance and their unit of measurement
+```
+Those who attend are constantly reminded the dangers of experimenting with electronics
+themselves without a full understanding of the possible consequences!
+```
+- Learn about microcontrollers (Arduino)
+- Learn to program an Arduino
+   * Use "sketch" to create programs
+   * Perform various hands on experiments using an Arduino
+- Create various projects using Arduino (following Dojo projects)
+- Conduct various mini projects throughout the year (2-3 weeks)
+- Introduce the Raspberry Pi
+   * Adventure through the world of physical computing with a Pi!
+- Work towards coming up with own project ideas and building a project as a team which could be entered into the Coder Dojo coolest project competition!
+
+## Pre-requisites
+- It is recommended that the course be undertaken by children 12 years and older
+- It is assumed kids will have previously attended Scratch/Web CoderDojo sessions
+- Have a willingness to learn and have fun :-)
+
+## When
+- The course is run during school term every Saturday from 11am-1pm.
+- Any Saturdays when the class isn't running will be announced in advance.
+
+## Labs
+Various hands on labs will be conducted throughout the year including:
+- Use "Hot wires" snap circuits to build circuits
+- Take apart a computer - understand various internal parts
+- Brining in old toys with motors
+- Create circuits
+- Tinkercad to create online circuits
+- Learn about Microcontrollers using arduino labs (Dojo sushis)
+    * Blinking LED
+    * Traffic Lights
+    * Temperature Sensors
+    * Ultrasonic Range Sensor
+    * LCDs
+    * Motors
+- Rasbperry Pi introduction
+  - Basic introduction to Rasbian
+  - Physical computing with Scratch (https://projects.raspberrypi.org/en/projects/physical-computing-with-scratch
+  - Finding your way around a Raspberry Pi and using a shell
+  - Basic linux commands
+  - Using Pi to perform physical computing
+
+## Group Projects
+
+- Kids can enter [Coolest Project](https://coolestprojects.org/)
+- Kids break up into groups come up with ideas for projects themselves. Alternatively perhaps mentors can
+assist coming up with a project.
+- Some assistance from parents or the club may be required if the team want to use equipment we don't currently have in stock
+- If the kids want to build something. Mentors will assist.
+
+## References / Resources
+
+* [Arduino resources](http://kata.coderdojo.com/wiki/Arduino_Path) Dojo Kata
+* [Raspberry Pi resources](https://curriculum.raspberrypi.org/physical-computing/creator/)
+* Coolect Projects [FAQ](https://help.coolestprojects.org/hc)
+* [Coder Dojo resources](https://coderdojo.com/resources/)
+
+## House rules
+
+- Kids under 13 must be accompanied by an adult
```

Comment:
This is great, can you check the page `en/about/class-information.html` and put the list there?

Reply:
It seems like the page `en/about/class-information.html` already has this content, so removing from `pages/en/partials/class-electronic.md` as per your request